lib/flutter_paytabs_bridge.dart

```diff
@@ -248,7 +248,7 @@ class FlutterPaytabsBridge {
 
   static Future<String> handleImagePath(String path) async {
     var bytes;
-    if (File(path).existsSync()) {
+    if (await File(path).exists()) {
       bytes = File(path).readAsBytesSync().buffer.asByteData();
     } else {
       bytes = await rootBundle.load(path);
```

Comment:
This will cause the same exception to be thrown.

Reply:
I think we should try the first commit as it is working as expected.